iron-query-params.html

```diff
@@ -52,10 +59,16 @@
       this._dontReact = false;
     },
 
+    detached: function() {
+      this.__ignoreLazy = false;
```

Comment:
Detaching and reattaching the element; it resets the internal / initial state, because without this line, if you detach and reattach it, it will overwrite the query params with whatever value it last had stored in memory.